Tests/TestingTests/RunnerTests.swift

```diff
@@ -459,7 +459,7 @@ final class RunnerTests: XCTestCase {
 #endif
 
     var configuration = Configuration()
-    configuration.deliverExpectationCheckedEvents = true
+    configuration.eventHandlingOptions.isExpectationCheckedEventEnabled = true
```

Comment:
As annoying as it is, can we add some stub test here somewhere that uses `deliverExpectationCheckedEvents` so we can get code coverage of it? Gotta keep those numbers up.

Reply:
Added